scipy/stats/stats.py

```diff
@@ -1599,6 +1599,7 @@ def normaltest(a, axis=0, nan_policy='propagate'):
 
     return NormaltestResult(k2, distributions.chi2.sf(k2, 2))
 
+Jarque_beratestResult = namedtuple('Jarque_beratestResult', ('statistic', 'pvalue'))
```

Comment:
Thanks @w-rfrsh. Can you please rename this `Jarque_beraResult`? The "test" is only in the nametuple name if it's also part of the corresponding function name (e.g. `skewtest`).

Reply:
Done, @rgommers. Also, I renamed the Shapiro's namedtuple in my PR #12056.